HealthcareSystem/GraphicalEditorServer/Controllers/EquipmentController.cs

```diff
@@ -16,35 +16,20 @@ namespace GraphicalEditorServer.Controllers
     [ApiController]
     public class EquipmentController : ControllerBase
     {
-        private readonly INonConsumableEquipmentService _nonConsumableEquipmentService;
-        private readonly IConsumableEquipmentService _consumableEquipmentService;
+        private readonly IEquipmentService _equipmentService;
 
-        public EquipmentController(INonConsumableEquipmentService nonConsumableEquipmentService, IConsumableEquipmentService consumableEquipmentService)
+        public EquipmentController(IEquipmentService equipmentService)
         {
-            _nonConsumableEquipmentService = nonConsumableEquipmentService;
-            _consumableEquipmentService = consumableEquipmentService; 
+            _equipmentService = equipmentService; 
         }
 
-        [HttpPost("consumable")]
-        public ActionResult AddConsumableEquipment([FromBody] String JSONString)
-        {
-            String JSONContent = StringToJSONFormat(JSONString);            
-
-            ConsumableEquipment cosumableEquipment = JsonConvert.DeserializeObject<ConsumableEquipment>(JSONContent);
-            _consumableEquipmentService.newConsumableEquipment(cosumableEquipment);
-
-            return Ok();
-        }
-
-        [HttpPost("nonconsumable")]
-        public ActionResult AddNonConsumableEquipment([FromBody] String JSONString)
+        [HttpPost]
+        public ActionResult AddEquipment([FromBody] String JSONString)
```

Comment:
Is there a reason why this is still like this? If you just left it as is and haven't tried switching to just having Equipment as the parameter, I'd think you should do that. If you have tried and it didn't work I'd like to see the error message you got.

Reply:
We are planning to add DTOs probably after adding GUI changes as a separate PR, so that is when we will try refactoring methods similar to this one.